src/main/java/org/manifold/compiler/front/ConstantValueVertex.java

```diff
@@ -0,0 +1,56 @@
+package org.manifold.compiler.front;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+import org.manifold.compiler.TypeValue;
+import org.manifold.compiler.Value;
+
+public class ConstantValueVertex extends ExpressionVertex {
+
+  private final Value value;
+  @Override
+  public Value getValue() {
+    return this.value;
+  }
+  public ConstantValueVertex (ExpressionGraph g, Value value) {
+    super(g);
+    this.value = value;
+  }
+
+  @Override
+  public void writeToDOTFile(BufferedWriter writer) throws IOException {
```

Comment:
A DOT file is a description of a graph as a diagram that can be processed by GraphViz. I think you mention this later, but using a visitor is kind of a better way to do this. Opened #47.